src/components/MdComponents.tsx

```diff
@@ -253,24 +252,22 @@ export const reactComponents = {
   ButtonLink,
   Card,
   ContentContainer,
-  DocLink,
   Divider,
+  DocLink,
   Emoji,
   ExpandableCard,
   GlossaryTooltip,
   InfoBanner,
   InfoColumn,
   InfoTitle,
   MobileButton,
-  QuizWidget,
   MobileButtonDropdown,
   Page,
+  QuizWidget,
   StyledButtonDropdown,
   Title,
   YouTube,
-  // Contributors,
-  // Logo,
-  // MeetupList,
+  Contributors,
```

Comment:
Can't ignore that this one is not ordered alphabetically 😆

Reply:
🤣